great-idea/index.html

```diff
@@ -17,7 +17,87 @@
   <body>
 
     <!-- Copy and paste your HTML from the first UI project here -->
+           
+    <div class="navigation">  
+      <nav>
+        <a href="#">Services</a>
+        <a href="#">Product</a>
+        <a href="#">Vision</a>
+        <a href="#">Features</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+      </nav>
+           <img class="logo" src="https://raw.githubusercontent.com/LambdaSchool/User-Interface/master/great-idea-website/img/logo.png" alt="Great Idea! Company logo.">
+   </div>
```

Comment:
Great job putting the image in that div. This saves alot of hassle when it comes to formatting the page and would be much more complicated had that not been the case.